apps/wallet/src/utils/hooks/navigation.ts

```diff
@@ -21,10 +21,21 @@ export const useNavigationHydrate = () => {
 			/* eslint-disable */
 			const findDashboard = (i: any) => i.name === 'Dashboard';
 			const dashboard = state?.routes.find(findDashboard)?.state;
-			const childRoute = dashboard?.routes[dashboard?.index as never];
-			const widget = childRoute?.params as any;
-			const widgetId = widget?.params?.id;
-			/* eslint-enable */
```

Comment:
keep eslint-enable or remove eslint-diable if not using `any` here

Reply:
@devtronghieu is that yours?